src/app/base/components/AppSidePanel/AppSidePanel.tsx

```diff
@@ -16,16 +16,14 @@ export type AppSidePanelProps = {
 
 const useCloseSidePanelOnRouteChange = (): void => {
   const { setSidePanelContent } = useSidePanel();
-  const history = useHistory();
+  const location = useLocation();
 
   // close side panel on route change
   useEffect(() => {
-    const unlisten = history.listen(() => setSidePanelContent(null));
-
     return () => {
-      unlisten();
+      setSidePanelContent(null);
     };
-  }, [history, setSidePanelContent]);
+  }, [location, setSidePanelContent]);
```

Comment:
What prompted this change? 

Previous version closes the panel immediately when the route changes.

Your updated implementation on the other hand relies on the effect's cleanup function to close the panel. This  means it closes the panel on the next effect run or unmount, not immediately upon route change. On top of this they may be small differences between the two that could lead to unexpected results. I'd try and keep the previous implementation if possible.

Reply:
Oh, I see, useHistory hook is no longer exported in the new version. You could create a new hook that uses our instance of redux history (setup in redux-store) to achieve the same thing.

Reply:
I've updated this to use the redux store history :+1: